### DISCUSSION TITLE
HELP et MODE D'EMPLOI

### DISCUSSION THREAD

**Discussion by Tournesol01** [A-S-T-U-C-E/STudio4Education#52] [Ideas]
Dans «Help», pourrait-on avoir accès à un lexique des commandes clavier du style «ctrl-?» ?

Pourrait-on avoir aussi un mode d'emploi qui explique les différentes fonctionnalités comme «Servers», «Wiring», «HTML factory...» ?

**Comment by SebCanet** [comment]
Si des volontaires se présentaient, j'en serais ravi :-D, mais en des années de développement, chacun a fait ses docs, wiki, sites, etc de son côté, pas de mutualisation hélas. Et comme c'est chronophage et que je suis débordé...
